lib/Util/Util.php

```diff
@@ -314,7 +314,7 @@ public static function flattenParamsList($value, $calculatedKey)
      */
     public static function urlEncode($key)
     {
-        $s = \urlencode($key);
+        $s = \urlencode((string) $key);
```

Comment:
Need to explicitly cast here because [`urlencode()`](https://www.php.net/manual/en/function.urlencode.php) expects strings.